Doc set for VMFR (#110937)
openshift/openshift-docs: modules/oadp-vmfr-accessing-files-ssh.adoc

// Module included in the following assemblies:
//
// backup_and_restore/application_backup_and_restore/oadp-vmfr/oadp-using-vmfr.adoc

:_mod-docs-content-type: PROCEDURE
[id="oadp-vmfr-accessing-files-ssh_{context}"]
= Accessing restored files through SSH

[role="_abstract"]

> Access restored virtual machine files through SSH by using `rsync`, `scp`, or `sftp` with the `VirtualMachineFileRestore` (VMFR) CR. This helps you to transfer files from VM backups efficiently.

Reviewer: ```suggestion
Access restored virtual machine (VM) files through SSH by using `rsync`, `scp`, or `sftp` with the `VirtualMachineFileRestore` (VMFR) custom resource (CR). You can transfer files from VM backups efficiently.
```

> When you configure SSH access, the VMFR controller automatically generates an SSH key pair and stores it in a Kubernetes secret. The default SSH username is `oadp`. The SSH file server listens on port `2222`.

Reviewer: ```suggestion
When you configure SSH access, the VMFR controller autogenerates an SSH key pair and stores it in a Kubernetes secret. The default SSH username is `oadp`. The SSH file server listens on port `2222`.
```

`automatically autogenerates` is a tautology
